### DISCUSSION TITLE
Add Positron to quarto get started page

### DISCUSSION THREAD

**Discussion by gshotwell** [posit-dev/positron#4290] [Feedback]
It might be good to add Positron to this page, even though it's in beta. The quarto experience in Positron is already significantly better than most of the listed editors. https://quarto.org/docs/get-started/

**Comment by juliasilge** [comment]
We definitely will be wanting to do this, yes! From my perspective, I am a little hesitant to do it TODAY but I think this fall (before the end of the year) is probably a good time to get Positron on that page.

**Reply by icarusz** [reply]
Agreed!  We will add it to the list and check in with the Positron team on timing!

**Reply by gshotwell** [reply]
As always I would just put it out there. There's plenty of warnings on the positron page, and Positron is head and shoulders above the other editors for working with Quarto.
